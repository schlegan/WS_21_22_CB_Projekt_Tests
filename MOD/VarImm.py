a = 96
b = -96
c = 19
d = -19
e = 0
print(a % 19) # 1
print(96 % c) # 1
print(0 % c) # 0
print(e % 19) # 0
print(b % 19) # -1
print(-96 % c) # -1
print(a % -19) # 1
print(96 % d) # 1
print(b % -19) # -1
print(-96 % d) # -1
print(0 % d) # 0
print(e % -19) # 0
