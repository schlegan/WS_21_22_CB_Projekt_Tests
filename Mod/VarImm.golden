1100-1-111-1-10092233720368547758079223372036854775807-1-100
